src/gameLogic/Game.tsx

```diff
@@ -35,6 +36,8 @@ import glassSound from '../assets/sounds/glass.mp3';
 import paperSound from '../assets/sounds/paper.mp3';
 import playerSprite from '../assets/player.png';
 
+import { TestFragment } from './fragments/testFragment';
+
 // const inputHandler = new InputHandler();
```

Comment:
🧹

Reply:
Is this related to my changes? Or you just want me to remove a comment that was previosly there?

Reply:
it's about the commented snippet of code. If you wish it to remain there just ignore my comment.